app/views/notifications/index.mobile.haml

```diff
@@ -24,4 +24,5 @@
                   .time_notif
                     = timeago(note.created_at)
 
-  = will_paginate @notifications, renderer: WillPaginate::ActionView::BootstrapLinkRenderer
+  = will_paginate @notifications, previous_label: "&laquo;", next_label: "&raquo;", inner_window: 1, outer_window: 0,
+  renderer: WillPaginate::ActionView::BootstrapLinkRenderer
```

Comment:
I'd prefer
```haml
  = will_paginate @notifications,
    previous_label: "&laquo;",
    next_label:     "&raquo;",
    inner_window:   1,
    outer_window:   0,
    renderer:       WillPaginate::ActionView::BootstrapLinkRenderer
```
Don't forget do run pronto afterwards since I didn't test this.